src/benchmarks/micro/corefx/System.Xml.Linq/Perf.XName.cs

```diff
@@ -13,22 +13,25 @@ public class Perf_XName
         private XName _noNamespace;
         private XName _hasNamespace;
 
+        [Benchmark]
+        public XName CreateElement() => XName.Get("node");
+
+        [Benchmark]
+        public XName CreateElementWithNamespace() => XName.Get("Root", "http://www.example.test");
+
+        [Benchmark]
+        public XName CreateElementWithNamespaceImplicitOperator() => "{Namespace}Root";
+
         [GlobalSetup]
         public void Setup()
         {
             _noNamespace = XName.Get("Root");
             _hasNamespace = XName.Get("{http://www.example.test}Root");
         }
 
-        [Benchmark]
-        public XName GetLocalName() => _noNamespace.LocalName;
```

Comment:
XName.LocalName not return XName object it return plain string, mistakenly set return type to XName which would run implicit constructor using the LocalName string. So this test has side effect and we don't need to test XName.LocalName as that will just return _localName field. So just removed it